xchange-bitstamp/src/main/java/info/bitrich/xchangestream/bitstamp/v2/BitstampStreamingService.java

```diff
@@ -52,12 +56,23 @@ protected void handleMessage(JsonNode message) {
         String channel = channelJsonNode.asText();
         String event = eventJsonNode.asText();
 
-        if (!channels.containsKey(channel)) {
-            LOG.warn("The message has been received from disconnected channel '{}'. Skipped.", channel);
-            return;
-        }
-        if (event.equals(JSON_DATA)) {
-            super.handleMessage(message);
+        switch (event) {
+            case EVENT_ORDERBOOK:
+            case EVENT_TRADE:
+                if (!channels.containsKey(channel)) {
+                    LOG.warn("The message has been received from disconnected channel '{}'. Skipped.", channel);
+                    return;
+                }
+                super.handleMessage(message);
+                break;
+            case EVENT_SUBSCRIPTION_SUCCEEDED:
+                LOG.info("Channel {} has been successfully subscribed", channel);
+                break;
+            case EVENT_UNSUBSCRIPTION_SUCCEEDED:
+                LOG.info("Channel {} has been successfully unsubscribed", channel);
+                break;
+            default:
+                LOG.error("Unsupported event type {} in message {}", event, message.toString());
```

Comment:
This might be too brittle. Is it really a problem if Bitstamp adds more events that you don't care about? My instinct would be to just ignore the event (maybe `log.debug`).

Reply:
May be "warn" level?

Reply:
Yeah, I can live with that.

Reply:
fixed